frontend/app-development/features/processEditor/handlers/OnProcessTaskAddHandler.test.ts

```diff
@@ -1,25 +1,81 @@
 import type { Policy } from 'app-shared/types/Policy';
 import type { OnProcessTaskEvent } from '@altinn/process-editor/types/OnProcessTask';
-import type { TaskEvent } from '@altinn/process-editor/utils/ProcessTaskManager';
 import { OnProcessTaskAddHandler } from './OnProcessTaskAddHandler';
+import { BpmnTypeEnum } from '@altinn/process-editor/enum/BpmnTypeEnum';
```

Comment:
We should avoid absolute imports until we have solved the import-issue within the team 😄